### ISSUE TITLE
PSR2.Methods.FunctionCallSignature broken with tab indentation

### ISSUE THREAD

**Issue by dereuromark** [squizlabs/PHP_CodeSniffer#3466]
```
 176 | ERROR | [x] Opening statement of multi-line function call not
     |       |     indented correctly; expected 0 spaces but found
     |       |     3
     |       |     (PSR2.Methods.FunctionCallSignature.OpeningIndent)
 177 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 7 spaces but found 4
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 178 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 7 spaces but found 4
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 179 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 7 spaces but found 4
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 180 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 7 spaces but found 4
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
```
on
```php
			// We only want cases from here on in.
			if ($type !== 'case') {
				continue;
			}

			$nextCode = $phpcsFile->findNext( // line 176
				T_WHITESPACE,
				$tokens[$nextCase]['scope_opener'] + 1,
				$nextCloser,
				true,
			);

```

Bug is visible in https://github.com/php-fig-rectified/psr2r-sniffer/runs/4141026695?check_suite_focus=true as well as the changes applied by fixer in https://github.com/php-fig-rectified/psr2r-sniffer/compare/show-bug

```
.../devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/ControlStructures/SwitchDeclarationSniff.php  5      0
.../data/www/git/psr2r-sniffer/PSR2R/Sniffs/ControlStructures/ConditionalExpressionOrderSniff.php  FAILED TO FIX
.../work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/ControlStructures/UnneededElseSniff.php  FAILED TO FIX
...rk/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/Commenting/DocBlockVarWithoutNameSniff.php  FAILED TO FIX
.../data/www/git/psr2r-sniffer/PSR2R/Sniffs/Commenting/FullyQualifiedClassNameInDocBlockSniff.php  FAILED TO FIX
/var/work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/Commenting/DocBlockParamSniff.php       FAILED TO FIX
...work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/WhiteSpace/UnaryOperatorSpacingSniff.php  FAILED TO FIX
/var/work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/Namespaces/UnusedUseStatementSniff.php  FAILED TO FIX
/var/work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/Classes/ClassCreateInstanceSniff.php    7      0
/var/work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/PHP/ListCommaSniff.php                  FAILED TO FIX
/var/work/devilbox/data/www/git/psr2r-sniffer/PSR2R/Sniffs/PHP/SingleQuoteSniff.php                4      0
```

The changes are breaking indentation completely. Also fails to fix, as then afterwards other sniffs are broken due to invalid changeset.

**Comment by jrfnl** [comment]
@dereuromark Did you mean to report this to the PSR2R repo ? Sounds to me more like a misconfigured ruleset than a problem with the sniff itself.
I use this sniff in tab-based codebases all the time without a problem.

**Comment by dereuromark** [comment]
It might be out of sync then as it extends the original one, but I will double check then, thanks.

**Comment by dereuromark** [comment]
No, it is actually using the latest repo version of this, so the error/fails look like a real bug.
In what way did you configure it to be working as expected?
